### ISSUE TITLE
Retry button on failed video does not work (then disappears)

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#6248]
The retry button doesn't seem to try to re-play the video. And then it disappears.

**Comment by JYamihud** [comment]
True. I find the same problem. Maybe a few second countdown thing could be added when a video isn't loading.

The user comes into the video, it fails to load, a countdown of say 5 seconds starts, when it ends it tries again to load the video. And so on until the video is either loaded or the user have canceled the loading. Also a button to reload immediately should be added.

**Comment by infinite-persistence** [comment]
Is this just due to the double-src problem?  Couldn't replicate with today's master.

**Comment by tzarebczan** [comment]
yea, seems to work now. That double src problem makes no sense lol

**Comment by tzarebczan** [comment]
This is happening again (possible due to double src problem being back)
